symbolic-ppdb/tests/test_ppdb.rs

```diff
@@ -73,3 +86,77 @@ fn test_embedded_sources_with_metadata_broken() {
     let items = iter.collect::<Result<Vec<_>, _>>().unwrap();
     assert_eq!(items.len(), 0);
 }
+
+#[test]
+fn test_matching_ids() {
+    let pdb_buf = std::fs::read(fixture("windows/portable.pdb")).unwrap();
+    let pdb = PortablePdb::parse(&pdb_buf).unwrap();
+    let pdb_debug_id = pdb.pdb_id().unwrap();
+
+    let pe_buf = std::fs::read(fixture("windows/integration.dll")).unwrap();
+    let pe = PeObject::parse(&pe_buf).unwrap();
+    let pe_debug_id = pe.debug_id();
+
+    assert_eq!(pe_debug_id, pdb_debug_id);
+}
+
+#[test]
+fn test_pe_embedded_ppdb_without_sources() {
+    let pe_buf = std::fs::read(fixture(
+        "windows/Sentry.Samples.Console.Basic-embedded-ppdb.dll",
+    ))
+    .unwrap();
+    let pe = PeObject::parse(&pe_buf).unwrap();
+
+    let embedded_ppdb = pe.embedded_ppdb().unwrap().unwrap();
+
+    let mut ppdb_buf = vec![0; embedded_ppdb.get_size()];
+    embedded_ppdb.decompress(&mut ppdb_buf).unwrap();
+    let ppdb = PortablePdb::parse(&ppdb_buf).unwrap();
+
+    assert!(ppdb.pdb_id().is_some());
```

Comment:
instead of just `is_some`, can we check it is the same `debug_id` as reported by the PE?